<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WPAEnterpriseIninputDlg</class>
 <widget class="QWidget" name="WPAEnterpriseIninputDlg">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>653</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_wifi_auth" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>681</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_userinfo" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_peap" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>500</width>
      <height>120</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>CA cert:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_peap_noCA_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Anony id:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_peap_dentity">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_peap_inner_auth">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>90</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Inner Auth:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>60</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PEAP Version:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_peap_version">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_peap_open_cacert">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_tls" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>500</width>
      <height>150</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>60</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>CA cert:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>User cert:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_tls_noCA">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Identity:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tls_identity">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>90</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Private Key</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>120</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tls_passwd">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tls_open_usercert">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tls_open_cacert">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tls_open_prikey">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_top" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Auth:               </string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_auth">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>5</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_fast" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>500</width>
      <height>120</height>
     </rect>
    </property>
    <widget class="QCheckBox" name="checkBox_fast_autoPAC">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string> Auto PAC:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_fast_PAC">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>221</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Anony id:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_fast_identity">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_fast_inner_auth">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>90</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Inner Auth:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>60</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PAC file:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_fast_open_pacfile">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_tunneled_tls" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>500</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>CA cert:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Anony id:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_tunneled_noCA">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tunneled_identity">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_tunneled_inner_auth">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>60</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Inner Auth:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tunneled_open_cacert">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_bottom" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>500</width>
      <height>25</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_ok">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
